env.default

```diff
@@ -28,6 +28,10 @@ AWS_S3_CUSTOM_DOMAIN=
 AWS_SECRET_ACCESS_KEY=
 AWS_STORAGE_BUCKET_NAME=
 
+
```

Comment:
Looks like there already is an empty line above. What do you think about removing this one?

Reply:
Great catch Tibor, updated!